File: Java/PB with Java/src/LandscapingOfYards.java
import java.util.Scanner;

public class LandscapingOfYards {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Area: ");
        int area = Integer.parseInt(scanner.nextLine());
        double price = (7.61 * area) - ((7.61 * area) * 0.18);
        double discount = ((7.61 * area) * 0.18);
        System.out.println("The final price is: " + price);
        System.out.println("The discount is: " + discount);
    }
}

File: C++/First Steps In Programming - Lab and Exc/LandscapingOfYards/LandscapingOfYards.cpp
#include <iostream>

int main() {
    int area;
    std::cout << "Area: ";
    std::cin >> area;

    double price = (7.61 * area) - ((7.61 * area) * 0.18);
    double discount = ((7.61 * area) * 0.18);

    std::cout << "The final price is: " << price << std::endl;
    std::cout << "The discount is: " << discount << std::endl;

    return 0;
}
